Dockerfile

```diff
@@ -1,53 +1,56 @@
 FROM centos:7
 ENV container docker
 
+# Get commonly used utilities
+RUN yum -y update && yum -y install -y wget which git deltarpm 
+
+# Install docker binaries 
+RUN yum install -y yum-utils device-mapper-persistent-data lvm2 && yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo && yum install -y docker-ce
+
+# Get Java
+RUN yum install -y java-1.8.0-openjdk java-1.8.0-openjdk-devel 
+
 # Add kbase user and set up directories
 RUN useradd -c "KBase user" -rd /kb/deployment/ -u 998 -s /bin/bash kbase && \
     mkdir -p /kb/deployment/bin && \
     mkdir -p /kb/deployment/jettybase/logs/ && \
     touch /kb/deployment/jettybase/logs/request.log && \
     chown -R kbase /kb/deployment
 
-# Get commonly used utilities
-RUN yum -y update && yum -y install -y wget which git deltarpm && \
-yum install -y java-1.8.0-openjdk java-1.8.0-openjdk-devel 
+#INSTALL DOCKERIZE
+RUN wget -N https://github.com/kbase/dockerize/raw/master/dockerize-linux-amd64-v0.6.1.tar.gz && tar xvzf dockerize-linux-amd64-v0.6.1.tar.gz && cp dockerize /kb/deployment/bin && rm dockerize*
+
 
+# Also add the user to the groups that map to "docker" on Linux and "daemon" on Mac
+RUN usermod -a -G 0 kbase && usermod -a -G 999 kbase
 
 # Install Condor
 RUN cd /etc/yum.repos.d && \
 wget http://research.cs.wisc.edu/htcondor/yum/repo.d/htcondor-development-rhel7.repo && \
 wget http://research.cs.wisc.edu/htcondor/yum/RPM-GPG-KEY-HTCondor && \
 rpm --import RPM-GPG-KEY-HTCondor && \
-yum -y install condor
+yum -y install condor-all
+
+# Install HTCondor Python Bindings
+RUN cd /root && \
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+    python get-pip.py && \
+    pip install htcondor  && \
+    rm /root/get-pip.py
 
-# Mount for cgroups
-VOLUME [ "/sys/fs/cgroup" ]
```

Comment:
This mount isn't needed?

Reply:
It doesn't do anything in the dockerfile.